common/src/thrift/tachyon.thrift

```diff
@@ -94,6 +95,41 @@ struct Command {
   2: list<i64> mData
 }
 
+struct LineageCommand {
+  1: CommandType mCommandType
```

Comment:
Why not do it in this PR? It would make more sense not to introduce inconsistency and it should not be a great deal of effort to take care of this.